### ISSUE TITLE
Uncaught Exception using the Scroll Controller plugin with a custom timeGridEvent component

### ISSUE THREAD

**Issue by toddb02** [schedule-x/schedule-x#288]
Using:
```
"@schedule-x/calendar": "1.19.0",
"@schedule-x/scroll-controller": "^1.19.0",
"@schedule-x/vue": "^1.15.0",
```

Using a custom component (or slot) for the timeGridEvent has an uncaught exception when using the ScrollController plugin.

This does not block rendering or break the plugin as far as I can tell but it is annoying to have the exception thrown constantly. 

<img width="773" alt="image" src="https://github.com/schedule-x/vue/assets/791004/652fb87c-9449-4c78-8397-187f5175043d">

[1709745357161.log](https://github.com/schedule-x/vue/files/14513436/1709745357161.log)

**Comment by tomosterlund** [comment]
Hey @toddb02, thanks for opening this issue.

I can confirm this on my end too, and will look into it. Most likely, the error throws on some kind of Fragment or Comment node injected by Vue, and I need to check for Node-Type before trying to read the `classList`.

**Comment by tomosterlund** [comment]
Just released in `@schedule-x/calendar@1.23.0`. Vue release will follow in a moment.

Thanks again :)
